stable/harbor/questions.yaml

```diff
@@ -0,0 +1,572 @@
+labels:
+  io.cattle.role: project # options are cluster/project
+questions:
+# Global Options
+- variable: harborAdminPassword
+  required: true
+  default: "Harbor12345"
```

Comment:
this reintroduces the hardcoded pw problem

Reply:
@dirkmueller yes, that is one way of looking at it, but Rancher has an option to generate the password in the UI

Reply:
@dirkmueller yep, it works fine if I don't specify a default value at all. Rancher won't let me continue unless I either generate a value or enter one manually